docs/navigating-compliance-json.md

```diff
@@ -0,0 +1,80 @@
+# Navigating Compliance Using Structured Data
+
+> Warning:
+This feature is in early development and is only implemented within the API.  There are multiple caveats that need to be considered and are listed below.
+
+## Caveats
+- Must have a `Platform` created specifically for the feature.
+- Both the compliance `feature` and `rule` need to be unique for the JSON `config-type`.
```

Comment:
Why is this again? Why not just the rule?

Reply:
In the ComplianceRule model, the unique_together constraint is feature and platform.

Reply:
Its just rule.  Will fix.